RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -203,7 +203,7 @@ boolean LX_islandAccess()
 		return LX_desertAlternate();
 	}
 
-	if((my_adventures() <= 9) || (my_meat() <= 1900))
+	if((my_adventures() <= 9) || (my_meat() < 1900))

```

Comment:
having exactly 1900 is enough. so it should be < not <=
although the odds of you having exactly 1900 and thus being short 1 single meat and triggering this return false are slim

Reply:
technically could be less in some paths or with discount giving skills